documentation/src/main/markdown/integrations/jenkinsplugin/autoescapingparameters.md

```diff
@@ -4,7 +4,7 @@ In [solution_name] for Jenkins, several special parameters are automatically esc
 The workflows pertaining to quotation marks and spaces are as follows.
 
 - Detect properties must be separated by spaces or carriage returns/line feeds.
-- Values containing spaces must be surrounded by either single or double quotation marks ('single' or “double”).
+- Values containing spaces must be surrounded by either single or double quotation marks ('single' or “double”) for Linux and Mac agents while for Windows you must use single quotes ('single').
```

Comment:
In the double quotes example the `“` character is shows. Should it be `"` instead?